scripts/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       target: swagger
     volumes:
       - transcendence-app-data:${TRANSCENDENCE_APP_DATA:-/var/lib/transcendence-app}
-      - ../transcendence-app/src:/transcendence-app/src
+      - ../transcendence-app/src:/transcendence-app/src:ro
```

Comment:
What does this `ro` mean?

Reply:
It means [read-only](https://docs.docker.com/compose/compose-file/#short-syntax-4) access.

More info here https://docs.docker.com/storage/volumes/#use-a-read-only-volume.